docs/jsonrpc_apis.md

```diff
@@ -606,35 +596,33 @@ results:
-| Port        | integer         | p2p network port                                            |
-| RPCPort     | integer         | json-RPC service port                                       |
-| RestPort    | integer         | RESTful service port                                        |
-| WSPort      | integer         | webservice port                                             |
-| OpenPort    | integer         | open service port                                           |
-| OpenService | bool            | if opens service enabled                                    |
-| Neighbors   | array[neighbor] | neighbor nodes information                                  |
+| compile     | string          | node's compile version                                      |
+| height      | integer         | current height of local node                                |
+| version     | integer         | peer-to-peer network protocol version of this node          |
+| services    | string          | the services provided by this node                          |
+| port        | integer         | P2P network port                                            |
+| rpcport     | integer         | JSON-RPC service port                                       |
+| restport    | integer         | RESTful service port                                        |
+| wsport      | integer         | webservice port                                             |
+| neighbors   | array[neighbor] | neighbor nodes information                                  |
 
 neighbor:
 
-| name       | type    | description                                                     |
-| ---------- | ------- | --------------------------------------------------------------- |
-| ID         | integer | neighbor's id                                                   |
-| HexID      | string  | neighbor's id in hex format                                     |
-| Height     | integer | neighbor current height                                         |
-| Services   | integer | neighbor service type. 4 is spv service and 0 is no spv service |
-| Relay      | bool    | whether neighbor will relay transaction or not                  |
-| External   | bool    | whether neighbor is from external network                       |
-| State      | string  | neighbor state in string format                                 |
-| NetAddress | string  | neighbor tcp address                                            |
+| name           | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| netaddress     | string  | network address of the neighbor in host:port format             |
+| services       | integer | neighbor service type. 4 is spv service and 0 is no spv service |
+| relaytx        | integer | relay transactions to the neighbor or not (0 true/1 false)      |
```

Comment:
Not matched to the result

Reply:
This PR is to change the result, you can't see the right result until this PR has merged.